src/main/java/gov/usgs/cida/pubs/webservice/AffliliationMvcService.java

```diff
@@ -195,7 +193,7 @@ public OutsideAffiliation createOutsideAffiliation(@RequestBody OutsideAffiliati
 		return result;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```

Comment:
These are commented out - can we get rid of them?

Reply:
I left the '@ApiOperation' lines in as comments as they will need to be converted to the new swagger if/when it is activated for secure endpoints.